DEBUG    twister_harness.device.factory:factory.py:37 Get device type "hardware"
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb90500d0>
DEBUG    httpcore.connection:_trace.py:85 start_tls.started ssl_context=<ssl.SSLContext object at 0xffffb8013f50> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:85 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7d79350>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:85 send_request_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Access-Control-Expose-Headers', b'Content-Range'), (b'Content-Length', b'188'), (b'Content-Range', b'projects 0-1/1'), (b'Content-Type', b'application/json'), (b'Date', b'Tue, 10 Dec 2024 21:23:21 GMT'), (b'Grpc-Metadata-Access-Control-Expose-Headers', b'Content-Range'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'0'), (b'X-Kong-Upstream-Latency', b'45')])
INFO     httpx:_client.py:1786 HTTP Request: GET https://api.golioth.io/v1/projects "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:85 response_closed.started
DEBUG    httpcore.http11:_trace.py:85 response_closed.complete
DEBUG    httpcore.connection:_trace.py:85 close.started
DEBUG    httpcore.connection:_trace.py:85 close.complete
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7f1e590>
DEBUG    httpcore.connection:_trace.py:85 start_tls.started ssl_context=<ssl.SSLContext object at 0xffffb7f08830> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:85 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb92c7d10>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:85 send_request_body.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Length', b'287'), (b'Content-Type', b'application/json'), (b'Date', b'Tue, 10 Dec 2024 21:23:21 GMT'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'0'), (b'X-Kong-Upstream-Latency', b'91')])
INFO     httpx:_client.py:1786 HTTP Request: POST https://api.golioth.io/v1/projects/firmware_ci/devices "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:85 response_closed.started
DEBUG    httpcore.http11:_trace.py:85 response_closed.complete
DEBUG    httpcore.connection:_trace.py:85 close.started
DEBUG    httpcore.connection:_trace.py:85 close.complete
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7eeded0>
DEBUG    httpcore.connection:_trace.py:85 start_tls.started ssl_context=<ssl.SSLContext object at 0xffffb7f08f80> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:85 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7d9a410>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:85 send_request_body.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Length', b'201'), (b'Content-Type', b'application/json'), (b'Date', b'Tue, 10 Dec 2024 21:23:22 GMT'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'2'), (b'X-Kong-Upstream-Latency', b'31')])
INFO     httpx:_client.py:1786 HTTP Request: POST https://api.golioth.io/v1/projects/firmware_ci/devices/6758b1498ab3217ea07d8b21/credentials "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:85 response_closed.started
DEBUG    httpcore.http11:_trace.py:85 response_closed.complete
DEBUG    httpcore.connection:_trace.py:85 close.started
DEBUG    httpcore.connection:_trace.py:85 close.complete
DEBUG    twister_harness.device.hardware_adapter:utils.py:37 Flashing command: /usr/local/bin/west flash --skip-rebuild --build-dir /__w/golioth-firmware-sdk/golioth-firmware-sdk/twister-out/esp32_devkitc_wrover_esp32_procpu/lightdb/delete/sample.golioth.lightdb_delete -- --esp-tool=/usr/local/bin/esptool.py --esp-device=/dev/serial/by-id/usb-Silicon_Labs_CP2102N_USB_to_UART_Bridge_Controller_342e9e0b1d28ee11811d0dd8f49e3369-if00-port0
DEBUG    twister_harness.device.hardware_adapter:hardware_adapter.py:132 Flashing finished
DEBUG    twister_harness.device.hardware_adapter:hardware_adapter.py:140 Opening serial connection for /dev/serial/by-id/usb-Silicon_Labs_CP2102N_USB_to_UART_Bridge_Controller_342e9e0b1d28ee11811d0dd8f49e3369-if00-port0
DEBUG    twister_harness.helpers.utils:utils.py:22 Found matching key: CONFIG_SHELL_PROMPT_UART="uart:~$ "
INFO     twister_harness.fixtures:fixtures.py:75 Wait for prompt
DEBUG    twister_harness.helpers.shell:shell.py:47 Got prompt
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7da6910>
DEBUG    httpcore.connection:_trace.py:85 start_tls.started ssl_context=<ssl.SSLContext object at 0xffffb8012cc0> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:85 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7da79d0>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:85 send_request_body.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Length', b'11'), (b'Content-Type', b'application/json'), (b'Date', b'Tue, 10 Dec 2024 21:23:40 GMT'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'1'), (b'X-Kong-Upstream-Latency', b'11')])
INFO     httpx:_client.py:1786 HTTP Request: POST https://api.golioth.io/v1/projects/firmware_ci/devices/6758b1498ab3217ea07d8b21/data/counter "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:85 response_closed.started
DEBUG    httpcore.http11:_trace.py:85 response_closed.complete
DEBUG    httpcore.connection:_trace.py:85 close.started
DEBUG    httpcore.connection:_trace.py:85 close.complete
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7da5210>
DEBUG    httpcore.connection:_trace.py:85 start_tls.started ssl_context=<ssl.SSLContext object at 0xffffb7f096d0> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:85 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7dca950>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:85 send_request_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Length', b'11'), (b'Content-Type', b'application/json'), (b'Date', b'Tue, 10 Dec 2024 21:23:40 GMT'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'0'), (b'X-Kong-Upstream-Latency', b'8')])
INFO     httpx:_client.py:1786 HTTP Request: GET https://api.golioth.io/v1/projects/firmware_ci/devices/6758b1498ab3217ea07d8b21/data/counter "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:85 response_closed.started
DEBUG    httpcore.http11:_trace.py:85 response_closed.complete
DEBUG    httpcore.connection:_trace.py:85 close.started
DEBUG    httpcore.connection:_trace.py:85 close.complete
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7da59d0>
DEBUG    httpcore.connection:_trace.py:85 start_tls.started ssl_context=<ssl.SSLContext object at 0xffffb7f091c0> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:85 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7da6450>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:85 send_request_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Access-Control-Expose-Headers', b'Content-Range'), (b'Content-Length', b'203'), (b'Content-Range', b'credentials 0-1/1'), (b'Content-Type', b'application/json'), (b'Date', b'Tue, 10 Dec 2024 21:23:43 GMT'), (b'Grpc-Metadata-Access-Control-Expose-Headers', b'Content-Range'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'1'), (b'X-Kong-Upstream-Latency', b'151')])
INFO     httpx:_client.py:1786 HTTP Request: GET https://api.golioth.io/v1/projects/firmware_ci/devices/6758b1498ab3217ea07d8b21/credentials "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:85 response_closed.started
DEBUG    httpcore.http11:_trace.py:85 response_closed.complete
DEBUG    httpcore.connection:_trace.py:85 close.started
DEBUG    httpcore.connection:_trace.py:85 close.complete
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[1;32muart:~$ [m[8D[J[1;32muart:~$ [msettings set golioth/psk-id generated-wYvVETkkKPGqBWFt@firmware_ci
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: Setting golioth/psk-id to generated-wYvVETkkKPGqBWFt@firmware_ci
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: Setting golioth/psk-id saved as generated-wYvVETkkKPGqBWFt@firmware_ci
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [msettings set golioth/psk generated-wYvVETkkKPGqBWFt
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: Setting golioth/psk to generated-wYvVETkkKPGqBWFt
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: Setting golioth/psk saved as generated-wYvVETkkKPGqBWFt
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [msettings set wifi/ssid "***TIMS_ORANGE_PI_WIFI_SSID***"
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: Setting wifi/ssid to ***TIMS_ORANGE_PI_WIFI_SSID***
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: Setting wifi/ssid saved as ***TIMS_ORANGE_PI_WIFI_SSID***
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [msettings set wifi/psk "***TIMS_ORANGE_PI_WIFI_PSK***"
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: Setting wifi/psk to ***TIMS_ORANGE_PI_WIFI_PSK***
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: Setting wifi/psk saved as ***TIMS_ORANGE_PI_WIFI_PSK***
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:08.498,000] [0m<inf> golioth_coap_client_zephyr: Golioth CoAP client connected[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:08.498,000] [0m<dbg> lightdb_delete: main: Before request (async)[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:08.498,000] [0m<dbg> lightdb_delete: main: After request (async)[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:08.498,000] [0m<inf> lightdb_delete: Golioth client connected[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:08.498,000] [0m<inf> golioth_coap_client_zephyr: Entering CoAP I/O loop[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:08.549,000] [0m<dbg> lightdb_delete: counter_handler: Counter deleted successfully[0m
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7dc9e10>
DEBUG    httpcore.connection:_trace.py:85 start_tls.started ssl_context=<ssl.SSLContext object at 0xffffb7f0a2a0> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:85 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7d975d0>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:85 send_request_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Length', b'13'), (b'Content-Type', b'application/json'), (b'Date', b'Tue, 10 Dec 2024 21:23:50 GMT'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'0'), (b'X-Kong-Upstream-Latency', b'11')])
INFO     httpx:_client.py:1786 HTTP Request: GET https://api.golioth.io/v1/projects/firmware_ci/devices/6758b1498ab3217ea07d8b21/data/counter "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:85 response_closed.started
DEBUG    httpcore.http11:_trace.py:85 response_closed.complete
DEBUG    httpcore.connection:_trace.py:85 close.started
DEBUG    httpcore.connection:_trace.py:85 close.complete
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7d97750>
DEBUG    httpcore.connection:_trace.py:85 start_tls.started ssl_context=<ssl.SSLContext object at 0xffffb7f0a600> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:85 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7dc4e50>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:85 send_request_body.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Length', b'11'), (b'Content-Type', b'application/json'), (b'Date', b'Tue, 10 Dec 2024 21:23:51 GMT'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'2'), (b'X-Kong-Upstream-Latency', b'10')])
INFO     httpx:_client.py:1786 HTTP Request: POST https://api.golioth.io/v1/projects/firmware_ci/devices/6758b1498ab3217ea07d8b21/data/counter "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:85 response_closed.started
DEBUG    httpcore.http11:_trace.py:85 response_closed.complete
DEBUG    httpcore.connection:_trace.py:85 close.started
DEBUG    httpcore.connection:_trace.py:85 close.complete
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7d968d0>
DEBUG    httpcore.connection:_trace.py:85 start_tls.started ssl_context=<ssl.SSLContext object at 0xffffb7f0ad50> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:85 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7dc7b90>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:85 send_request_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Length', b'11'), (b'Content-Type', b'application/json'), (b'Date', b'Tue, 10 Dec 2024 21:23:51 GMT'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'0'), (b'X-Kong-Upstream-Latency', b'10')])
INFO     httpx:_client.py:1786 HTTP Request: GET https://api.golioth.io/v1/projects/firmware_ci/devices/6758b1498ab3217ea07d8b21/data/counter "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:85 response_closed.started
DEBUG    httpcore.http11:_trace.py:85 response_closed.complete
DEBUG    httpcore.connection:_trace.py:85 close.started
DEBUG    httpcore.connection:_trace.py:85 close.complete
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[1;32muart:~$ [m[8D[J[00:00:13.498,000] [0m<dbg> lightdb_delete: main: Before request (sync)[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:13.551,000] [0m<dbg> lightdb_delete: counter_delete_sync: Counter deleted successfully[0m
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7dbb350>
DEBUG    httpcore.connection:_trace.py:85 start_tls.started ssl_context=<ssl.SSLContext object at 0xffffb7f0b2f0> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:85 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7dc61d0>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:85 send_request_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Length', b'13'), (b'Content-Type', b'application/json'), (b'Date', b'Tue, 10 Dec 2024 21:23:55 GMT'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'0'), (b'X-Kong-Upstream-Latency', b'8')])
INFO     httpx:_client.py:1786 HTTP Request: GET https://api.golioth.io/v1/projects/firmware_ci/devices/6758b1498ab3217ea07d8b21/data/counter "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:85 response_closed.started
DEBUG    httpcore.http11:_trace.py:85 response_closed.complete
DEBUG    httpcore.connection:_trace.py:85 close.started
DEBUG    httpcore.connection:_trace.py:85 close.complete
DEBUG    twister_harness.device.hardware_adapter:hardware_adapter.py:187 Closed serial connection for /dev/serial/by-id/usb-Silicon_Labs_CP2102N_USB_to_UART_Bridge_Controller_342e9e0b1d28ee11811d0dd8f49e3369-if00-port0
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:85 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7d99410>
DEBUG    httpcore.connection:_trace.py:85 start_tls.started ssl_context=<ssl.SSLContext object at 0xffffb8013c80> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:85 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0xffffb7dda4d0>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.started request=<Request [b'DELETE']>
DEBUG    httpcore.http11:_trace.py:85 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:85 send_request_body.started request=<Request [b'DELETE']>
DEBUG    httpcore.http11:_trace.py:85 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.started request=<Request [b'DELETE']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Length', b'2'), (b'Content-Type', b'application/json'), (b'Date', b'Tue, 10 Dec 2024 21:23:56 GMT'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'1'), (b'X-Kong-Upstream-Latency', b'71')])
INFO     httpx:_client.py:1786 HTTP Request: DELETE https://api.golioth.io/v1/projects/firmware_ci/devices/6758b1498ab3217ea07d8b21 "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.started request=<Request [b'DELETE']>
DEBUG    httpcore.http11:_trace.py:85 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:85 response_closed.started
DEBUG    httpcore.http11:_trace.py:85 response_closed.complete
DEBUG    httpcore.connection:_trace.py:85 close.started
DEBUG    httpcore.connection:_trace.py:85 close.complete